templates/zerver/apps.html

```diff
@@ -66,4 +66,35 @@ <h2>Apps for every platform.</h2>
         </div>
     </div>
 </div>
+
+<script>
```

Comment:
The JS Code should go in `\static\js`, more specifically `\static\js\click_handlers.js` would be a good place move this code to.